README.md

```diff
@@ -1,68 +1,79 @@
-<p align="center">
-<img src="/logo.png" alt="butler" style="max-width:100%;">
-</p>
+![butler](/logo.png)
```

Comment:
The image should be centered.

Reply:
If it must be then we need to keep the HTML. I will update.

Reply:
Yep, it looks nicer 😄